cli/magic-config.ts

```diff
@@ -68,6 +70,9 @@ const embeddingModels = [
       options.enableSagemakerModels = config.llms?.sagemaker
         ? config.llms?.sagemaker.length > 0
         : false;
+      options.enableSagemakerModelsSchedule = config.llms?.sagemakerSchedule?.enabled;
```

Comment:
Can you test for the presence of `config.llms?.sagemakerSchedule` first, and then check the properties within that? I'm thinking of a scenario here I say set start and end times but don't set enabled = true ... or is that a scenario this should support? If so ignore this.